src/runner.py

```diff
@@ -171,47 +158,54 @@ def get_info(self):
 
     def reconcile(self, virt_type, quantity):
         """Bring runners in line with target."""
-        active_runners = []
-        stale_runners = []
-        for runner in self.get_info():
-            if runner.virt_type == virt_type:
-                if runner.is_active:
-                    active_runners.append(runner)
-                else:
-                    stale_runners.append(runner)
+        runners = [r for r in self.get_info() if r.virt_type == virt_type]
+        online_runners = [r for r in runners if r.is_online and r.is_local]
```

Comment:
The `r.is_local` condition ensures that the RunnerManager operates only on runners that belong to its unit.

Reply:
agree.  In fact this is 
```suggestion
        local_online_runners = [r for r in runners if r.is_online and r.is_local]
```